### ISSUE TITLE
Typo on documentation?

### ISSUE THREAD

**Issue by Scoodood** [keras-team/keras#9821]
The [documentation](https://keras.io/backend/#backend-functions) of `keras.backend.gradients(loss, variables)` said

`Returns the gradients of variables w.r.t. loss.`

But I think it should be 

`Returns the gradients of loss w.r.t. variables`
right?

**Comment by gabrieldemarmiesse** [comment]
Seems reasonable, can you do a PR for this?

**Comment by Scoodood** [comment]
I tried to fork this repo to my account so that I can create a new branch, make changes and the do pull request. But the forking process seems to stuck forever... any idea how to fix it? Any better way to do PR?

![screen shot 2018-04-04 at 12 38 41 am](https://user-images.githubusercontent.com/15989924/38294762-c6e73474-37a0-11e8-8851-bc266d996fe3.png)

**Comment by gabrieldemarmiesse** [comment]
wow, that's some bad luck right there. I'll do this PR when I have the time then, since it's not urgent. Thanks for reporting it!

**Comment by tiferet** [comment]
Looks like this issue should be closed, by PR #9866.

**Comment by gabrieldemarmiesse** [comment]
Thanks!
